ambassador/ambassador/envoy/v2/v2listener.py

```diff
@@ -509,6 +510,38 @@ def __init__(self, config: 'V2Config', listener: IRListener) -> None:
                     'format': 'ACCESS [%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\"\n'
                 }
             } ]
+             # Use sane access log spec in JSON
+            if Config.envoy_log_format == "json" :
+                self.access_log = [ {
+                    'name': 'envoy.file_access_log',
+                    "config": {
```

Comment:
Looking at the rest of the file, it looks like single quotes are preferred to double quotes - let's make them all single quotes?
And re formatting, it'd be good to also keep to pep8 as much as possible, e.g. always have a space after colons.